src/models/Forecast.js

```diff
@@ -0,0 +1,105 @@
+export default class Forecast {
+  constructor(data) {
+    this.data = data;
+  }
+
+  createForecast() {
+    const forecastSection = document.querySelector('.forecast');
+    forecastSection.innerHTML = '';
+
+    Object.values(this.data).map((dayForecast, index) => {
+      const date = new Date(dayForecast.datetime);
+
+      const forecastBlockHtml = `
+        <div class="forecast-block">
+          <div class="forecast-block__content">
+            <h3 class="forecast-block__date">
+              ${this.getWeekDay(date)} ${date.getDate()}
+            </h3>
+            <div class="forecast-block__icon-container">
+              <img class="forecast-block__icon">
+            </div>
+            <span class="forecast-block__temp">
+              ${dayForecast.min_temp}°C - ${dayForecast.max_temp}°C
+            </span>
+          </div>
+        </div>
+      `;
+      forecastSection.insertAdjacentHTML('beforeend', forecastBlockHtml);
+
+      const forecastIcon = forecastSection.querySelectorAll('.forecast-block__icon')[index];
+      forecastIcon.src = require(`../images/weather-icons/${dayForecast.weather.icon}.png`);
+
+      const forecastBlock = forecastSection.querySelectorAll('.forecast-block')[index];
```

Comment:
I would better build this HTML using `document.createElement` and so on.
Because the way you do is not optimized:
1. create and inject block
2. query all icons and select the one you need by index
3. query all blocks and select the one you need by index

You do a lot of extra DOM maniulations, you need to minimize them:

1. create and inject block should be enough
```js
const block = document.createElement('div')
block.className = 'forecast-block'
// ...
const icon = document.createElement('img')
icon.src = require(`../images/weather-icons/${dayForecast.weather.icon}.png`);

block.addEventListener('click', () => { ... })
```